app/controllers/cart_controller.rb

```diff
@@ -0,0 +1,18 @@
+class CartController < ApplicationController
+  def create
+    if Cart.last
+      cart = Cart.last
+    else
+      cart = Cart.create
+    end
+    photo = Photo.find(params[:photo_id])
+    
+    if cart.add(photo, photo.price)
+      flash[:notice] = photo.title + " was  successfully added to cart!"
+      redirect_to photos_path
```

Comment:
Follow what @houndci-bot is telling you here... You're using the `if` statement to set the flash message and you're redirecting to the same path so you might as well remove the `redirect_to` statement outside of the `if` block